packages/util/obojobo-lib-utils/package.json

```diff
@@ -47,8 +47,8 @@
 		"jest": "24.1.0",
 		"lint-staged": "^6.1.1",
 		"prettier": "1.11.1",
-		"react-dom": "16.7.0",
-		"react-test-renderer": "16.7.0",
+		"react-dom": "16.8.6",
```

Comment:
I believe we shouldn't have react-dom here under devDeps since its already under regular deps.